app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -652,30 +632,27 @@ class PipelineSampleReport extends React.Component {
         }
       },
       () => {
+        // TODO (gdingle): do we really want to reload the page here?
```

Comment:
What is the alternative?

Reply:
The alternative is to change only the URL with history.pushState. 

Another question about this call is why only set the background_id? Why not keep all current param values?